v2/internal/internalgauntlettest/gauntlet_test.go

```diff
@@ -190,30 +200,51 @@ func TestGauntlet(t *testing.T) {
 
 				for _, encoding := range encodings {
 					t.Run(encoding, func(t *testing.T) {
-
-						var resHeaders map[string]string
-						callOptions := newCallOptions(&resHeaders)
-
-						// call with encoding clients
-						switch encoding {
-						case _json:
-							validateJSON(t, client, callOptions)
-						case _thrift:
-							validateThrift(t, client, callOptions)
-						case _proto:
-							validateProto(t, client, callOptions)
-						default:
-							t.Fatalf("unsupported encoding %s", encoding)
-						}
-
-						// validate response headers
-						wantResponseHeaders := map[string]string{
-							_headerKeyRes: _headerValueRes,
-						}
-						assert.Equal(t, wantResponseHeaders, resHeaders, "response headers did not match")
+						runConcurrent(t, concurrency, client, encoding)
 					})
 				}
 			}
 		})
 	}
 }
+
+// runConcurrent runs the test 'concurrency' times
+func runConcurrent(t *testing.T, concurrency int, client yarpc.Client, encoding string) {
+	var wg sync.WaitGroup
+	start := make(chan struct{})
+
+	for i := 0; i < concurrency; i++ {
+		wg.Add(1)
+		go func() {
+			<-start
```

Comment:
I ran a test. This <-start close(start) pattern is counter-productive and actually increases the start time variance of each of the goroutines.

Reply:
:O interesting, I'll drop the channel